src/main/scala/com/github/unchama/seichiassist/subsystems/vote/subsystems/fairy/domain/property/FairyEndTime.scala

```diff
@@ -0,0 +1,5 @@
+package com.github.unchama.seichiassist.subsystems.vote.subsystems.fairy.domain.property
+
+import java.time.LocalDateTime
+
+case class FairyEndTime(endTimeOpt: Option[LocalDateTime])
```

Comment:
```suggestion
case class FairyEndTime(endTimeOpt: LocalDateTime)
```

妖精を召喚した時点で終わりの時間は自ずと確定するのでOptionである必要性はありません

Reply:
これ、そうなんですが、もともとの実装でSQL上に`VARCHAR`型で時間を文字列保存していたので、今回の再実装で`DATETIME`型に変更して、もともとの値をNULLに置き換えたのでOptionじゃないとエラーになると思います。